connections/Result.php

```diff
@@ -3,7 +3,11 @@
 namespace Connections;
 
 /**
- * database query result.
+ * Database query result.
+ * @method array<int, array<string, string>> fetchAll(...$args) fetches all
+ * rows.
+ * @method array<string, string> fetchAssoc() fetches one row in the form of
+ * associative array.
```

Comment:
Right now only these two methods are needed. However, if there is a need to add something else later it can be done with ease.